sdk/storage/README.md

```diff
@@ -0,0 +1,19 @@
+
+Azure Storage is a Microsoft-managed service providing cloud storage that is highly available, secure, durable, scalable, and redundant. Azure Storage includes Blobs (objects), Queues, and Files.
+
+## Getting started
+
+To get started with a specific library, see the **README.md** file located in the library's project folder. You can find service libraries in the `/sdk/storage/azure-storage-<subcomponent>` directory.
+- [Azure Storage Blob](azure-storage-blob/README.md) is Microsoft's object storage solution for the cloud. Blob storage is optimized for storing massive amounts of unstructured data that does not adhere to a particular data model or definition, such as text or binary data.
+- [Azure Storage Blob Batch](azure-storage-blob-batch/README.md) allows you to batch multiple Azure Blob Storage operations in a single request.
+- [Azure Storage Blob Cryptography](azure-storage-blob-cryptography/README.md) supports client-side encryption for Azure Storage block blobs.
+- [Azure Storage Queue](azure-storage-queue/README.md) is a service for storing large numbers of messages.  A queue message can be up to 64 KB in size and a queue may contain millions of messages, up to the total capacity limit of a storage account.
+- [Azure Storage Common](azure-storage-common/README.md) provides infrastructure shared by the other Azure Storage client libraries like shared key authentication and exceptions.
+- [Azure Storage File Share](azure-storage-file-share/README.md) service shares are accessible via the Server Message Block (i.e. SMB) protocol, and also via REST APIs. The File Share service offers the following four resources: the storage account, shares, directories, and files. Shares provide a way to organize sets of files and also can be mounted as an SMB file share that is hosted in the cloud.
+- [Azure Storage File Datalake](azure-storage-file-datalake/README.md) is Microsoft's optimized storage solution for for big data analytics workloads. A fundamental part of Data Lake Storage Gen2 is the addition of a hierarchical namespace to Blob storage. The hierarchical namespace organizes objects/files into a hierarchy of directories for efficient data access. 
+
```

Comment:
Looking good! In the directory I also see Queue Cryptography, Common, management ones, and microsoft-azure-storage-blob? should they be described too?

Reply:
I have already explained 'Azure Storage Common'  and others you mentioned are not in this folder, what you see is just a copy of old versions. And  we do not need to explain them.

Reply:
So they are going to be deleted? Maybe I am checking something wrong but V11 is still there?

Reply:
I need to find out from team why we are keeping 6-7 older version of azure-mgmt-storage in this location. And based on that I will inform you what are we going to do with this.

Reply:
Thanks! Please do - if they are to stay there we should mention what they are.